src/core/connection.c

```diff
@@ -207,6 +207,32 @@ QuicConnAlloc(
             SourceCid->CID.Length,
             SourceCid->CID.Data);
 
+        if (Session->Settings.ServerResumeOrZeroRtt) {
```

Comment:
This is never going to be set on server, as the connection is allocated before the real session is assigned.

Reply:
I'd like to avoid allocating the memory for the transport parameters, only to free it immediately later.
Do you have suggestions on how to work around this?  Moving the allocation to someplace is possible, but seems hacky.

Reply:
I moved the allocation to when the session claims the connection. The allocation fails silently and sending resumption will fail silently on that connection.